charts/argo/values.yaml

```diff
@@ -4,7 +4,7 @@ images:
   server: argocli
   executor: argoexec
   pullPolicy: Always
-  tag: v2.7.6
+  tag: v2.9.0-rc2
```

Comment:
I would leave this out, we don't want the chart to deploy RC2 by default, if someone wants to use RC2 they should just set the tag themselves.

Reply:
Done.